tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -458,6 +459,7 @@ def test_pantsd_invalidation_stale_sources(self):
     finally:
       rm_rf(test_path)
 
+  @unittest.skip("Pantsd is not expected to handle parallel runs anymore.")
```

Comment:
Is this long-term true? I thought we did expect pantsd to still handle parallel runs of some goals, including list?

If it's long-term true, let's delete the test.
If it's not long-term true, let's add a link to a ticket about fixing it.

Reply:
thanks, added TODO